Gems/LmbrCentral/Code/Source/Audio/EditorAudioPreloadComponent.cpp

```diff
@@ -25,6 +25,11 @@ namespace LmbrCentral
                 ->Field("Load Type", &EditorAudioPreloadComponent::m_loadType)
                 ;
 
+            serializeContext->Enum<AudioPreloadComponent::LoadType>()
```

Comment:
hmmm, we should probably document this somewhere if this is a new requirement...

Reply:
Yeah we should note it wherever we are planning to have DPE documentation. This is the least used variant of all the different ways we support reflecting enum type display values. Most of the variants involve specifying the values in either a list, or generic values, or with EnumAttributes. This is the only place where you specify only the enum as a data element and rely on the values to be reflected separately, which for whatever reason only these 4 fields only did it in the EditContext, whereas the other several ones did it in both places, so it's one of those "I'm not actually sure how it was working before in the RPE."